bindings/python/scripts/parse.py

```diff
@@ -120,24 +119,22 @@ def generate_parsed_info(node):
     cursor = node["cursor"]
     depth = node["depth"]
 
-    # @TODO: fix
-    if cursor.spelling != "":
```

Comment:
Were there any unnamed nodes which were giving you an issue becase of which you turned this check on in the first place? Or was this accidental and in error?

Reply:
Yeah, previous clang version.
Some cursors were `Nonetype`, thus they didn't have the `spelling` attribute.